Library/Formula/pjscrape.rb

```diff
@@ -0,0 +1,22 @@
+require "formula"
+
+class Pjscrape < Formula
+  homepage "https://nrabinowitz.github.io/pjscrape/"
+  url "https://github.com/nrabinowitz/pjscrape/archive/v0.1.4.tar.gz"
+  sha256 "a605781448486dfaeeceba1fdf5884170ebe385c776d7c5719b2f559d6a74ba6"
+
+  head "https://github.com/nrabinowitz/pjscrape.git"
+
+  depends_on "phantomjs"
+
+  def install
+    libexec.install Dir["*"]
+
+    sh = bin/"pjscrape"
+    sh.write <<-EOS.undent
```

Comment:
Can probably just do ` (bin/"pjscrape").write <<-EOS.undent ` here, since the binary name isn't too long.